### PR TITLE
nmsderp

### PR DESCRIPTION
**Pull request by nmsderp**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x ] The file is in the `domains` folder and is in the JSON format.
- [x ] You have completed your website. 
- [x ] The website is reachable.  
- [x ] You're not using Vercel or Netlify.  
- [x ] The CNAME record doesn't contain `https://` or `/`.  
- [x ] There is sufficient information at the `owner` field.  

## Website Link/Preview
https://nmsderp.github.io

### CODE REVIEW

File: domains/nmsderp.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A place for my small projects, and maybe a blog in the future.",
+    "repo": "https://github.com/nmsderp/nmsderp.github.io",
+    "owner": {
+        "username": "nmsderp",
+        "email": "nmsderp@gmail.com",
+        "twitter": "i don't have a twitter lol"
```
Comment (Omxpro) [inline domains/nmsderp.json:7]: if you dont have twitter then please remove the field. other then that all is good

**Review by Omxpro — CHANGES_REQUESTED** [review]

File: domains/nmsderp.json (hunk 1 above)
Comment (nmsderp) [reply domains/nmsderp.json:7]: alr

**Comment by nmsderp** [comment]
@Omxpro fixed it

**Review by Omxpro — APPROVED** [review]
good.

File: domains/nmsderp.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "A place for my small projects, and maybe a blog in the future.",
+    "repo": "https://github.com/nmsderp/nmsderp.github.io",
+    "owner": {
+        "username": "nmsderp",
+        "email": "nmsderp@gmail.com",
```
Comment (CuteDog5695) [inline domains/nmsderp.json:6]: remove comma at line 6

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by nmsderp** [comment]
@CuteDog5695 fixed

**Review by iostpa — APPROVED** [review]
now seems good to me

**Pull request merged**